tests/Handler/Image/InfoTest.php

```diff
@@ -146,4 +146,17 @@ public function testSvgFromFile()
         $this->assertSame('image/svg+xml', $info->getMime());
         $this->assertInstanceOf(Image\SvgType::class, $info->getType());
     }
+
+    public function testSvgWithoutXmlDeclaration()
+    {
+        $file = $this->filesystem->getFile('fixtures/images/nut.svg');
+        $data = $file->read();
+        $data = substr($data, 39);
```

Comment:
I was trying not to add another file, because it throws off all the finder / iterator tests which list all the files.